src/codehelp/migrations/20250112--codehelp--add_query_limits.sql

```diff
@@ -0,0 +1,16 @@
+-- SPDX-FileCopyrightText: Mark Liffiton <liffiton@gmail.com>, Rana Moeez Hassan
+--
+-- SPDX-License-Identifier: AGPL-3.0-only
+
+PRAGMA foreign_keys = OFF;
```

Comment:
I don't think this script does anything that would affect foreign keys, so the foreign keys off/on can be removed to simplify it.